src/kontrol/foundry.py

```diff
@@ -260,9 +260,10 @@ def custom_view(self, contract_name: str, element: KCFGElem) -> Iterable[str]:
     def build(self) -> None:
         try:
             run_process(['forge', 'build', '--build-info', '--root', str(self._root)], logger=_LOGGER)
-        except FileNotFoundError:
-            print("Error: 'forge' command not found. Please ensure that 'forge' is installed and added to your PATH.")
-            sys.exit(1)
```

Comment:
On the other hand, I agree with this change: `sys.exit` should only be called from `__main__` (or the CLI layer in general).